schema.json

```diff
@@ -0,0 +1,18 @@
+[
+  // Enter the schema here. For example:
+  //
+  //{
+  //  "namespace": "hello", // For example: browser.hello.*
+  //  "description": "Hello world",
+  //
+  // "permissions": ["experiments.simple"], // Should start with "experiments"
```

Comment:
And I think you just explained why my test experiment I was trying out today failed... thanks.